modules/protocol/chatircd1.1.c

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright (c) 2014-2017 ChatLounge IRC Network Development Team
+ * Copyright (c) 2003-2004 E. Will et al.
+ * Copyright (c) 2005-2007 Atheme Development Group
+ *
+ * Rights to this code are documented in doc/LICENSE.
+ *
+ * This file contains protocol support for ChatIRCd, a charybdis fork.
+ * Adapted from the Charybdis protocol module.
+ */
+
+#include "atheme.h"
+#include "uplink.h"
+#include "pmodule.h"
+#include "protocol/chatircd.h"
+
+DECLARE_MODULE_V1
+(
+	"protocol/chatircd", true, _modinit, NULL,
+	PACKAGE_STRING,
+	"ChatLounge IRC Network Development Team <http://www.chatlounge.net>"
```

Comment:
https perhaps?

Reply:
Nope, this should be `VENDOR_STRING`, as in all other modules. Since it is in fact, a vendor, not a developer string, Atheme is the appropriate content for this.